api/validator.js

```diff
@@ -27,9 +27,10 @@ const SCHEMA_JOB_PERMUTATION = Joi.object()
         description: Job.description,
         environment: Job.environment,
         image: Job.image,
+        requires: Job.requires,
         secrets: Job.secrets,
         settings: Job.settings,
-        requires: Job.requires
+        sourcePaths: Job.sourcePaths
```

Comment:
Did we decide to have a separate keyword for this? or just use `settings`?

Reply:
separate keyword/job config, rather than in settings. #911 has details.